GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift

```diff
@@ -182,9 +192,9 @@ extension SecureConversations {
         }
 
         func renderProps() {
-            header.title = props.style.headerTitle
-            header.backButton.tap = props.backButtonTap.execute
-            header.closeButton.tap = props.closeButtonTap.execute
+            header.props.title = props.style.headerTitle
```

Comment:
I would suggest to have Header.Props be formed in WelcomeViewModel along with WelcomeView.Props, to avoid mutation on-fly. So in result there would be something like: ` header.props = props.headerProps`.

Reply:
fixed - 4513a5708d31ec0fd4c0bde57b25f7a2786f7ff6